tests/Database/Traits/EncryptableTest.php

```diff
@@ -22,7 +22,7 @@ public function setUp(): void
     public function testEncryptableTrait()
     {
         $testModel = new TestModelEncryptable();
-        $testModel->setEncrypter($this->encrypter);
+        $testModel->encryptUsing($this->encrypter);
```

Comment:
This should probably remain using the existing interface since that's the public API for the encryptable trait

Reply:
Yes, but I think the encryptable traight might need to be modified, as laravel now supports encrypting attributes.

https://github.com/illuminate/database/blob/master/Eloquent/Concerns/HasAttributes.php#L131
https://github.com/illuminate/database/blob/master/Eloquent/Concerns/HasAttributes.php#L934

using something like

```php
<?php

namespace App;

use App\Casts\EncryptCast;
use Illuminate\Database\Eloquent\Model;

class User extends Model
{
    protected $casts = [
        'age' => EncryptCast::class,
    ];

    // ...
}
```
I had set the trait to leverage the use of the static encrypter variable https://github.com/wintercms/storm/pull/26/files#diff-b89596612b5ccf11dfa4953e4290a2723f7972adf31aa9d9ee47efdaf500f9c0R106 instead of the local variable.

But that should probably happen its own focussed pr?, I can revert it for now if you want.

Reply:
Yeah, let's do that. I don't feel comfortable mashing them together at the moment, that would be better reviewed in a separate PR with it's own discussion.

Reply:
It should be reset now back to the original implementation. The only change is the variable name due to the variable name conflict with the static variable in the parent class.

Reply:
Ah, gotcha, that makes my other comment redundant. Thanks!